network/utils_test.go

```diff
@@ -79,3 +81,31 @@ func TestSafeCloseConnection(t *testing.T) {
 		t.Fatal("Close method return incorrect error")
 	}
 }
+
+func TestGetDBURLHost(t *testing.T) {
+	t.Run("MySQL valid connection URL", func(t *testing.T) {
+		url := "test:test@tcp(localhost:3306)/test"
+		host, err := GetDBURLHost(url, true)
+		assert.NoError(t, err)
+		assert.Equal(t, "localhost", host)
+	})
+
+	t.Run("MySQL invalid connection URL", func(t *testing.T) {
+		url := "test:test@tcp://localhost:3306/test"
```

Comment:
lets test `test:test@tcp:localhost:3306/test` and `test:test@localhost:3306/test`  values with `useMysql=True`. And some PostgreSQL specific URLs. 
And MySQL specific strings with `useMysql=false` to test reverse conditions.